storage/local/chunk/chunk.go

```diff
@@ -402,23 +409,23 @@ func newIndexAccessingChunkIterator(len int, acc indexAccessor) *indexAccessingC
 	return &indexAccessingChunkIterator{
 		len:       len,
 		pos:       -1,
-		lastValue: ZeroSamplePair,
+		lastValue: model.SamplePair{Timestamp: model.Earliest},
```

Comment:
Ok, will do - and to be consistent, also move `ZeroSample` along with `ZeroSamplePair` to model?